packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.Props.ts

```diff
@@ -75,6 +75,22 @@ export interface ICheckboxProps extends React.ButtonHTMLAttributes<HTMLElement |
   theme?: ITheme;
 
   /**
+   * Accessible label for the checkbox. Defaults to 'label' property value.
+   */
+  ariaLabel?: string;
+
+  /**
+   * ID for element that contains label information for the checkbox.
+   * Defaults to 'label' property value.
```

Comment:
I would prefer we leave out any mention of default behavior here. It's unnecessary and potentially confusing. The default behavior is that everything just works without further intervention from the user.